src/main/java/org/jitsi/videobridge/xmpp/MediaStreamTrackFactory.java

```diff
@@ -174,8 +178,14 @@ else if (temporalIdx > 0)
                     rtpEncodings[idx]
                         = new RTPEncodingDesc(track, idx,
                         primary[streamIdx],
-                        rtx == null || rtx.length == 0 ? -1 : rtx[streamIdx],
                         temporalId, spatialId, height, frameRate, dependencies);
+                    SecondarySsrcs ssrcSecondarySsrcs = secondarySsrcs.get(primary[streamIdx]);
```

Comment:
There are some hidden NPEs here as the encoding expects a `long` and we have a `Long`. In the current flows I don't think we will ever get an NPE, but then again I've seen a impossible exception in the bridge logs..

Reply:
Yeah, it seems like we shouldn't ever hit that in this flow (I think `SourcePacketExtension#getSSRC` would have to return an invalid `Long` for that to happen?)  Is there another scenario where it could?